platform/barefoot/rules.mk

```diff
@@ -23,4 +24,4 @@ SONIC_ALL += $(SONIC_ONE_IMAGE) $(SONIC_ONE_ABOOT) \
 $(LIBSAIREDIS)_DEPENDS += $(BFN_SAI) $(WNC_OSW1800_PLATFORM) $(BFN_INGRASYS_PLATFORM) $(BFN_PLATFORM)  #$(LIBSAITHRIFT_DEV_BFN)
 
 # Runtime dependency on sai is set only for syncd
-$(SYNCD)_RDEPENDS += $(BFN_SAI) $(WNC_OSW1800_PLATFORM) $(BFN_INGRASYS_PLATFORM) $(BFN_PLATFORM)
+$(SYNCD)_RDEPENDS += $(BFN_MODULE) $(BFN_SAI) $(WNC_OSW1800_PLATFORM) $(BFN_INGRASYS_PLATFORM) $(BFN_PLATFORM)
```

Comment:
Do we need this?

Reply:
Isn't the presence of these modules mandatory at runtime for syncd to run?